internal/config/dash.go

```diff
@@ -94,49 +97,59 @@ type Dash interface {
 	Validate() error
 
 	ModuleManager() module.ManagerInterface
+
+	// ClusterClientManager()
```

Comment:
If this is going to be implemented in a future change, create a todo?

Reply:
It is, it is, just a left over comment. Line 22 is identical (and uncommented)